data/json/monsters/reptile_amphibian.json

```diff
@@ -390,5 +390,103 @@
     "anger_triggers": [ "STALK", "PLAYER_WEAK", "PLAYER_CLOSE" ],
     "flags": [ "SEES", "SMELLS", "HEARS", "SWIMS" ],
     "armor": { "bash": 4 }
+  },
+  {
+    "id": "mon_gecko",
+    "type": "MONSTER",
+    "name": { "str": "gecko" },
+    "description": "The gecko is a small reptile related to the lizard. It is usually medium-small in size and does not exceed 16 centimeters in length, including the tail",
+    "default_faction": "small_animal",
+    "species": [ "REPTILE" ],
+    "diff": 1,
+    "volume": "100 ml",
+    "weight": "100 g",
+    "hp": 6,
+    "speed": 120,
+    "material": [ "flesh" ],
+    "symbol": "g",
+    "color": "brown",
+    "aggression": -50,
+    "morale": 60,
+    "aggro_character": false,
+    "melee_skill": 5,
+    "melee_dice": 1,
+    "melee_dice_sides": 4,
+    "melee_damage": [ { "damage_type": "cut", "amount": 4 } ],
+    "dodge": 3,
+    "vision_day": 30,
+    "vision_night": 5,
+    "dissect": "dissect_lizard_sample_single",
+    "anger_triggers": [ "HURT" ],
+    "fear_triggers": [ "PLAYER_CLOSE" ],
+    "placate_triggers": [ "PLAYER_WEAK" ],
+    "upgrades": { "half_life": 34, "into": "mon_gecko_big" },
+    "flags": [ "SEES", "HEARS", "SMELLS", "HARDTOSHOOT", "SWIMS" ],
+    "reproduction": { "baby_egg": "egg_gecko", "baby_count": 8, "baby_timer": 26 }
+  },
+  {
+    "id": "mon_gecko_big",
+    "type": "MONSTER",
+    "name": { "str": "big gecko" },
+    "description": "A mutant bullfrog the size of a person. Better not get too close.",
+    "default_faction": "mutant",
+    "bodytype": "lizard",
+    "species": [ "REPTILE" ],
+    "volume": "6250 ml",
+    "weight": "81500 g",
+    "hp": 70,
+    "speed": 110,
+    "material": [ "flesh" ],
+    "symbol": "G",
+    "color": "green",
+    "aggression": 10,
+    "morale": 100,
+    "aggro_character": false,
+    "melee_skill": 5,
+    "melee_dice": 1,
+    "melee_dice_sides": 12,
+    "melee_damage": [ { "damage_type": "cut", "amount": 1 } ],
+    "dodge": 3,
+    "harvest": "mutant_animal_large_noskin",
+    "dissect": "dissect_batrachian_sample_small",
+    "path_settings": { "max_dist": 5 },
+    "upgrades": { "half_life": 60, "into": "mon_gecko_mega" },
+    "zombify_into": "mon_zombie_gecko",
+    "anger_triggers": [ "STALK", "PLAYER_WEAK", "PLAYER_CLOSE" ],
+    "flags": [ "SEES", "SMELLS", "HEARS", "SWIMS" ],
+    "armor": { "bash": 4, "electric": 1 }
+  },
+  {
+    "id": "mon_gecko_mega",
+    "type": "MONSTER",
+    "name": { "str": "colossal gecko" },
+    "description": "A gecko bloated to titanic size. Run.",
+    "default_faction": "mutant",
+    "bodytype": "lizard",
+    "species": [ "REPTILE" ],
+    "volume": "92500 ml",
+    "weight": "120 kg",
+    "hp": 800,
+    "speed": 100,
+    "material": [ "flesh" ],
+    "symbol": "G",
+    "color": "green",
+    "aggression": 5,
+    "morale": 100,
+    "aggro_character": false,
+    "melee_skill": 4,
+    "melee_dice": 6,
+    "melee_dice_sides": 10,
```

Comment:
Consider special attack instead of bump damage - a bite and maybe a tailslap instead of just generic hits.